### PR TITLE
New overweight scenario (half_overweight)

### PR DESCRIPTION
**Pull request by bodirsky**
## :bird: Description of this PR :bird:

- add a less extreme scenario for the reduction of overweight (now overweight and obesity is halved compared to the baseline)

## :wrench: Checklist for PR creator :wrench:

- [ X] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [ X] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [ X] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [X] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `Rscript start.R --> "compilation check"`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
    - Default run from the PR target branch for comparison
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : ** mins
  - This PR's default :  ** mins

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [ ] content review done (at least 1)
- [x] RSE review done (at least 1)

### CODE REVIEW

File: config/default.cfg [hunk 1]
```diff
@@ -407,9 +407,11 @@ cfg$gms$s15_exo_diet <- 0               # def = 0
 # * only activated if s15_exo_diet is set to 1
 # *   options:   healthy_BMI, 2100kcal, 2500kcal,
 # *              endo, no_underweight, no_overweight
+# *              half_overweight, no_underweight_half_overweight
 # * healthy_BMI: all people have a BMI of 20-25
 # * no_underweight: all people with BMI<20 have a BMI 20-25
 # * no_overweight: all people with BMI>25 have a BMI 20-25
+# * half_overweight: halfe baseline people with BMI>25 have a BMI 20-25
```
Comment (tscheypidi) [inline config/default.cfg:414]:
```suggestion
# * half_overweight: halved amount of people with BMI>25 compared to baseline (have BMI 20-25 instead)
```

**Review by tscheypidi — APPROVED** [review]
description of the new setting should be a bit clearer (see my suggestion), otherwise ok

File: config/default.cfg (hunk 1 above)
Comment (FelicitasBeier) [reply config/default.cfg]: Description to half_overweight:
- number of people overweight is halved (halbiert)
- half of the people with BMI>25 have a BMI 20-25 (Hälfte der Leute)

File: modules/15_food/anthro_iso_jun22/exodietmacro.gms [hunk 2]
```diff
@@ -273,6 +273,40 @@ $elseif "%c15_kcal_scen%" == "no_overweight"
...
+               + p15_bmi_shr_calibrated(t,iso,sex,age,"mediumhigh")/2
+               + p15_bmi_shr_calibrated(t,iso,sex,age,"high")/2
+               + p15_bmi_shr_calibrated(t,iso,sex,age,"veryhigh")/2
+               + p15_bmi_shr_calibrated(t,iso,sex,age,"medium");
+     p15_bmi_shr_target(t,iso,sex,age,"verylow")=0;
+     p15_bmi_shr_target(t,iso,sex,age,"low")=0;
+     p15_bmi_shr_target(t,iso,sex,age,"mediumhigh")=p15_bmi_shr_calibrated(t,iso,sex,age,"mediumhigh")/2;
+     p15_bmi_shr_target(t,iso,sex,age,"high")=p15_bmi_shr_calibrated(t,iso,sex,age,"high")/2;
+     p15_bmi_shr_target(t,iso,sex,age,"veryhigh")=p15_bmi_shr_calibrated(t,iso,sex,age,"veryhigh")/2;
+     i15_intake_scen_target(t,iso)$(sum((sex,age), im_demography(t,iso,sex,age)) >0 ) =
```
Comment (FelicitasBeier) [inline modules/15_food/anthro_iso_jun22/exodietmacro.gms:305]: i15_intake_scen_target seems to be the same across scenarios. Is it necessary to double the code or could that be outside the if?

**Review by FelicitasBeier — APPROVED** [review]
Update of description & double-check whether doubling of code is necessary. Otherwise I would approve.

**Review by weindl — APPROVED** [review]
Looks good.

File: modules/15_food/anthro_iso_jun22/exodietmacro.gms (hunk 2 above)
Comment (bodirsky) [reply modules/15_food/anthro_iso_jun22/exodietmacro.gms:305]: no, there are also other elseifs

**Pull request merged**